internal/olm/client/client.go

```diff
@@ -103,22 +103,30 @@ func (c Client) DoDelete(ctx context.Context, objs ...runtime.Object) error {
 		}
 		kind := obj.GetObjectKind().GroupVersionKind().Kind
 		log.Infof("  Deleting %s %q", kind, getName(a.GetNamespace(), a.GetName()))
-		err = c.KubeClient.Delete(ctx, obj, client.PropagationPolicy(metav1.DeletePropagationForeground))
+		err = c.KubeClient.Delete(ctx, obj, client.PropagationPolicy(metav1.DeletePropagationBackground))
```

Comment:
Because of this change we might want to test installing both OLM latest and the most recent pre-latest (0.14.1?) version.

Reply:
I actually tried a slightly different implementation of this that worked with 0.14.1 but did not work with 0.10.1.  So I left the test as is (check latest and 0.10.1) to make sure it at least works with a current version and an older version.

Reply:
Sounds good.